src/cbc_sdk/platform/audit.py

```diff
@@ -81,6 +84,23 @@ def get_auditlogs(cb):
         res = cb.get_object("/integrationServices/v3/auditlogs")
         return res.get("notifications", [])
 
+    @staticmethod
+    def get_queued_auditlogs(cb):
+        """
+        Retrieve queued audit logs from the Carbon Black Cloud server.
+
+        Required Permissions:
+            org.audits (READ)
+
+        Args:
+            cb (BaseAPI): Reference to API object used to communicate with the server.
+
+        Returns:
+            list[AuditLog]: List of objects representing the audit logs, or an empty list if none available.
+        """
+        res = cb.get_object(AuditLog.urlobject.format(cb.credentials.org_key) + "/_queue")
+        return [AuditLog(cb, -1, data) for data in res.get("results", [])]
```

Comment:
Instead of putting negative one why don't we remove the model_unique_id from the init and set -1 in the super call

Reply:
You could've mentioned this in the _previous_ ticket. :)

Reply:
I missed it before